controller/projects.go

```diff
@@ -151,6 +151,13 @@ func (p *Projects) Get(id uuid.UUID) (*model.Project, error) {
 		return nil, errors.Wrap(err, "failed to get project")
 	}
 
+	if proj.TransactionExecutionCount == 0 {
```

Comment:
Is this only relevant for migrated projects when they are first accessed? if so would it make more sense to put this inside migrations https://github.com/onflow/flow-playground-api/blob/master/migrate/migrate.go****

Reply:
I see. OK.

Reply:
Just add comment explaining why is that there/